Tests/SourceGenerators/SourceGeneratorAssemblyToProcess/ErrorClasses/UnsupportedBacklink.cs

```diff
@@ -0,0 +1,42 @@
+﻿////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2022 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License")
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+using System.Linq;
+using Realms;
+
+namespace SourceGeneratorPlayground
+{
+    public partial class UnsupportedBacklink : IRealmObject
+    {
+        [Backlink("WrongPropertyName")]
+        public IQueryable<BacklinkObj> WrongBacklinkProp { get; }
+
+        [Backlink(nameof(BacklinkObj.InverseLink))]
+        public IQueryable<BacklinkObj> CorrectBacklinkProp { get; }
+
+        public IQueryable<string> IgnoredBacklink { get; }
+    }
+
+    public partial class BacklinkObj : IRealmObject
+    {
+        public int Id { get; set; }
+
+        [PrimaryKey]
```

Comment:
Why is this applied on the link property?

Reply:
The expectation here is that all the classes in this folder are generating only diagnostics and no actual code. This was to be sure that this class would not be generated.